db/migrate/20160606093754_add_tftp_management_to_template_kind.rb

```diff
@@ -0,0 +1,16 @@
+class AddTftpManagementToTemplateKind < ActiveRecord::Migration
+  def change
+    add_column :template_kinds, :tftp_management, :boolean, :default => false
```

Comment:
It's a leftover from my previous generation patch. Removing:

    scope :with_tftp, -> { where(:tftp_management => true).order(:name) }

And the migration itself:

```ruby
class AddTftpManagementToTemplateKind < ActiveRecord::Migration
  def change
    add_column :template_kinds, :tftp_management, :boolean, :default => false

    reversible do |dir|
      dir.up do
        # seed script was not yet executed - create explicitly
        TemplateKind.create(:name => "PXEGrub2")
        TemplateKind.where(:name => %w(PXELinux PXEGrub PXEGrub2 POAP ZTP)).update_all(:tftp_management => true)
      end
      dir.down do
        TemplateKind.delete_all(:name => "PXEGrub2")
      end
    end
  end
end
```